simglucose/analysis/report.py

```diff
@@ -93,16 +93,20 @@ def percent_stats(BG, ax=None):
 
 
 def risk_index_trace(df_BG, visualize=False):
-    chunk_BG = [df_BG.iloc[i:i + 60, :] for i in range(0, len(df_BG), 60)]
+    chunk_BG = [df_BG.iloc[i:i + 20, :] for i in range(0, len(df_BG), 20)]
```

Comment:
Why changing 60 to 20? Maybe you can make it an input parameter to allow customization of the step size?